docs/USAGE.md

```diff
@@ -0,0 +1,282 @@
+# Usage
+
+**WARNING**: Please notice that no official release has been created for this repo yet, so installation instructions below would require you to build the code manually beforehand.
+
+## Configure the OpenTelemetry Tracing Library for .NET
+
+### Configuration values
+
+Use these environment variables to configure the tracing library:
+
+| Environment variable | Default value | Description |
+|-|-|-|
+| `OTEL_SERVICE_NAME` |  | The name of the service. |
+| `OTEL_ENV` |  | The value for the `environment` tag added to every span. This determines the environment in which the service is available in OTEL µAPM.  |
+| `OTEL_TRACE_ENABLED` | `true` | Enable to activate the tracer. |
+| `OTEL_TRACE_DEBUG` | `false` | Enable to activate debugging mode for the tracer. |
+| `OTEL_TRACE_AGENT_URL` | `http://localhost:9080/v1/trace` | The hostname and port for a OTEL Smart Agent or OpenTelemetry Collector. |
+| `OTEL_TRACE_GLOBAL_TAGS` |  | Comma-separated list of key-value pairs to specify global span tags. For example: `"key1:val1,key2:val2"` |
+| `OTEL_LOGS_INJECTION` | `false` | Enable to inject trace IDs, span IDs, service name and environment into logs. This requires a compatible logger or manual configuration. |
+| `OTEL_MAX_LOGFILE_SIZE` | `10 MB` | The maximum size for tracer log files, in bytes. |
+| `OTEL_TRACE_LOG_PATH` | Linux: `/var/log/OTEL/dotnet/dotnet-profiler.log`<br>Windows: `%ProgramData%"\OTEL .NET Tracing\logs\dotnet-profiler.log` | The path of the profiler log file. |
+| `OTEL_DIAGNOSTIC_SOURCE_ENABLED` | `true` | Enable to generate troubleshooting logs with the `System.Diagnostics.DiagnosticSource` class. |
+| `OTEL_DISABLED_INTEGRATIONS` |  | The integrations you want to disable, if any, separated by a semi-colon. These are the supported integrations: AspNetMvc, AspNetWebApi2, DbCommand, ElasticsearchNet5, ElasticsearchNet6, GraphQL, HttpMessageHandler, IDbCommand, MongoDb, NpgsqlCommand, OpenTracing, ServiceStackRedis, SqlCommand, StackExchangeRedis, Wcf, WebRequest |
```

Comment:
Could you please double-check if it is really `;` ? I remember being mentioned in some other PR that we should standardized to `,`. If currently is semi-colon please open an issue to fix it.

Reply:
Yes, this uses `;`:
`internal static class ConfigurationSourceExtensions
    {
        private const char Separator = ';';`

Reply:
@RassK FYI

Reply:
@dszmigielski resolving the review, please do not forget to create the issue.